test/e2e/install_test.go

```diff
@@ -27,103 +27,147 @@ const (
 var _ = Describe("Operator Install", func() {
 	var (
 		ctx             context.Context
-		pkgName         string
 		operatorName    string
 		operator        *operatorv1alpha1.Operator
 		operatorCatalog *catalogd.Catalog
 	)
 	When("An operator is installed from an operator catalog", func() {
 		BeforeEach(func() {
 			ctx = context.Background()
-			pkgName = "prometheus"
-			operatorName = fmt.Sprintf("operator-%s", rand.String(8))
-			operator = &operatorv1alpha1.Operator{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: operatorName,
-				},
-				Spec: operatorv1alpha1.OperatorSpec{
-					PackageName: pkgName,
-				},
-			}
-			operatorCatalog = &catalogd.Catalog{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: testCatalogName,
-				},
-				Spec: catalogd.CatalogSpec{
-					Source: catalogd.CatalogSource{
-						Type: catalogd.SourceTypeImage,
-						Image: &catalogd.ImageSource{
-							Ref: testCatalogRef,
-						},
-					},
-				},
-			}
-		})
-		It("resolves the specified package with correct bundle path", func() {
-			err := c.Create(ctx, operatorCatalog)
+			var err error
+			operatorCatalog, err = createTestCatalog(ctx, testCatalogName, testCatalogRef)
```

Comment:
Why does the operator catalog need to be created before each spec?

Reply:
You're right that it doesn't have to be created before each spec. I can move it to the `BeforeSuite()` process to create  the catalog before the test suite even runs and then clean it up in an `AfterSuite()`

Reply:
https://github.com/operator-framework/operator-controller/pull/242/commits/95771f9a2f470e3a21c4f309995ed3d00cdf6525 makes it so the catalog is created in `BeforeSuite()` and cleaned up in `AfterSuite()`